hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/io/FlinkMergeHandle.java

```diff
@@ -138,12 +128,12 @@ public void rollOver(Iterator<HoodieRecord<T>> newRecordsItr) {
     this.writeStatus.setTotalErrorRecords(0);
     this.timer = new HoodieTimer().startTimer();
 
-    rollNumber++;
+    rollNumber += 1;
```

Comment:
What's the difference between these changes?

Reply:
No difference, just make the logic more clear ~